tests/end_to_end/test_helper.cpp

```diff
@@ -677,39 +677,32 @@ _test_helper_end_to_end::~_test_helper_end_to_end()
 
         // Run down duplicate handles, if any.
         _duplicate_handles.rundown();
-    } catch (Catch::TestFailureException&) {
-    }
 
-    // Verify that all maps were successfully removed.
-    uint32_t id;
-    if (!ebpf_fuzzing_enabled) {
-        REQUIRE(bpf_map_get_next_id(0, &id) < 0);
```

Comment:
Deliberately removed this check:

1. Since maps handles can be closed aynchronously in a work-item, there is no guarantee all maps have been deleted by this time.
2. There are some tests which pin and unpin objects, but in fault injection, they do not unpin the objects. That causes rundown ref leak, hence we cannot wait for work item rundown to go to 0 here.
3. If any object is leaked, memory leak detection will anyways catch the leak.